pybamm/install_odes.py

```diff
@@ -8,28 +8,32 @@
 
 from pybamm.util import root_dir
 
-try:
-    # wget module is required to download SUNDIALS or SuiteSparse.
-    import wget
+def check_platform():
+    if sys.platform == "win32":
+        raise Exception("pybamm_install_odes is not supported on Windows.")
 
-    NO_WGET = False
-except ModuleNotFoundError:
-    NO_WGET = True
+check_platform()
 
+def install_required_module(module):
+    try:
+        __import__(module)
+    except ModuleNotFoundError:
+        print(f"{module} module not found. Installing {module}...")
+        subprocess.run(["pip", "install", module], check=True)
```

Comment:
Curious about the `__import__` syntax here since I have seen that this is offered and used more for programmatic ways of importing modules as objects. Could you try using the `importlib` module for this instead? I found this thread insightful: https://stackoverflow.com/questions/28231738/import-vs-import-vs-importlib-import-module

Reply:
Done 👍🏻